driver-core/src/main/com/mongodb/client/model/Projections.java

```diff
@@ -57,6 +58,10 @@ private Projections() {
         return new SimpleExpression<TExpression>(fieldName, expression);
     }
 
+    public static <TExpression> Bson computed(Map<String, TExpression> fields) {
```

Comment:
This lets you specify a bunch of computed fields without having to use `combine`